sklearn/mixture/bayesian_mixture.py

```diff
@@ -781,3 +779,34 @@ def _set_parameters(self, params):
                                       self.precisions_cholesky_.T)
         else:
             self.precisions_ = self.precisions_cholesky_ ** 2
+
+    def partial_fit(self, X, y=None):
+        """Estimate model parameters with the EM algorithm using the posteriors
+        of previous fits as priors.
+
+        The method fit the model `n_init` times and set the parameters with
+        which the model has the largest likelihood or lower bound. Within each
+        trial, the method iterates between E-step and M-step for `max_iter`
+        times until the change of likelihood or lower bound is less than
+        `tol`, otherwise, a `ConvergenceWarning` is raised.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            List of n_features-dimensional data points. Each row
+            corresponds to a single data point.
+
+        Returns
+        -------
+        self
+        """
+        if hasattr(self, 'converged_'):
+            self.covariance_prior = self.covariances_[0]
+            self.mean_prior = self.mean_prior_
+            self.mean_precision_prior = self.mean_precision_prior_
+            self.weight_concentration_prior = self.weight_concentration_prior_
+            self.degrees_of_freedom_prior = self.degrees_of_freedom_prior_
```

Comment:
you cannot modify init param during fit or partial_fit. It's against the sklearn API. 

I would create a private _fit function that takes at input X and all the initial values of parameters.

clear?